workflow-job

```diff
@@ -10,6 +10,7 @@ import org.kohsuke.github.GHRepository
 // $repoURL parameter : specifies the commiter GitHub repo URL
 // $sha1 parameter : specifies the actual commit
 
+def dockerIp = '62.210.101.42'
```

Comment:
Shouldn't that be passed from env

Reply:
yes it should but we have issues passing this env to the slave, so we don't use it for now

Reply:
I think I don't understand the difference with docker_host